plugin/audit_log/audit_log.c

```diff
@@ -186,16 +187,29 @@ void escape_buf(const char *in, size_t *inlen, char *out, size_t *outlen,
   while ((in < inend) && (out < outend))
   {
     replaced= FALSE;
-    for (rule= escape_rules; rule->character; rule++)
-    {
-      if (*in == rule->character)
-      {
+    if ((unsigned char)(*in) < 32) {
```

Comment:
Why a typecast is needed here?

Reply:
I could eliminate this cast by specifying less than space, but I'm not sure if that would be clearer.